app/helpers/bulk_download_types_helper.rb

```diff
@@ -14,12 +15,18 @@ module BulkDownloadTypesHelper
   RESQUE_EXECUTION_TYPE = "resque".freeze
   VARIABLE_EXECUTION_TYPE = "variable".freeze
   ECS_EXECUTION_TYPE = "ecs".freeze
+  MANUAL_EXECUTION_TYPE = "manual".freeze
```

Comment:
Can we call this `MANUAL_UPLOAD_TYPE` since the admin is just uploading a file, and nothing is being executed at all?
Also add a comment explaining that "Bulk downloads with this type are not auto-generated and the kickoff function will do nothing. An admin operator will need to manually upload a file to the s3 location designated by download_output_key before calling mark_success."

Reply:
Good point, I made the change and added the comment